beacon-chain/monitor/service.go

```diff
@@ -104,8 +104,6 @@ func NewService(ctx context.Context, config *ValidatorMonitorConfig, tracked []t
 
 // Start sets up the TrackedValidators map and then calls to wait until the beacon is synced.
 func (s *Service) Start() {
-	s.Lock()
-	defer s.Unlock()
 
```

Comment:
Why is this one removed?

Reply:
It seems the analyzer doesn't distinguish between spawned go routines and normal ones